app/models/pieces/pawn.rb

```diff
@@ -1,2 +1,22 @@
 class Pawn < Piece
+  # rubocop:disable all
+  def valid_move?(x_destination, y_destination)
+    # check for Piece valid_move? conditions
+    return false unless super(x_destination, y_destination)
+    # a pawn can never move horizontally or, and never move more than two fields ahead, or more than one sideways
+    return false if moving_horizontally?(y_destination) || diff(x_destination, x_position) > 1 || diff(y_destination, y_position) > 2
```

Comment:
You can probably eliminate the `diff(y_destination, y_position) > 2` in favor of the explicit check later (except using `>=` instead).